tests/e2escenarios/e2e_images_test.go

```diff
@@ -130,23 +131,27 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the openjdk-11-rhel8 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", "openjdk/openjdk-11-rhel8:latest", "java:8", "openjdk-11-rhel8")
-			verifySupportedImage("openjdk/openjdk-11-rhel8:latest", "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
+			redhatOpenjdk12RHEL8Project := util.GetEnvWithDefault("REDHAT_OPENJDK11_RHEL8_PROJECT", "openjdk-11-rhel8")
+			oc.ImportImageFromRegistry("registry.redhat.io", "openjdk/openjdk-11-rhel8:latest", "java:8", redhatOpenjdk12RHEL8Project)
+			verifySupportedImage("openjdk/openjdk-11-rhel8:latest", "openjdk", "java:8", redhatOpenjdk12RHEL8Project, appName, commonVar.Context)
```

Comment:
Can we please do away with using `redhat` and `RHEL` in variable names? At other places in the file as well.

Reply:
I see little point. The name is already there in code `registry.redhat.io` for example. I think as much as possible, envs names should make clear, the intent of use.
cc @girishramnani @kadel wdyt